src/pudl/transform/eia860.py

```diff
@@ -860,6 +860,15 @@ def clean_emissions_control_equipment_eia860(
         "emission_control_equipment_cost",
     ] = 3200
 
+    # Fix bad operational status code. Decided to fix it here instead of in the
+    # code_fixes portion of the CODE_METADATA because it felt more like a one-off
+    # rather than something that should be applied across the board. I determined that
+    # this was the right fix by looking at the operational_status_code for this
+    # emissions control unit in the surrounding yeras.
```

Comment:
What's the argument for not integrating this into the general encoding system we've already got? `CS` is an invalid code, and the only reason this wouldn't work in general is if it we discovered that it showed up somewhere else with a different meaning.  Do you know what `CS` stands for here or why it showed up rather than one of the expected codes?

Reply:
I think I agree with @zaneselvans that using the existing encoding system would be preferable unless there's a specific reason that might be a problem. However, if we decide not to do that I think it would be better to have a more targeted spot-check here. As it is now, if `CS` were to show up again in the future we would quietly replace it when that might not be what we want.

Reply:
I think @aesharpe's idea here is that the information about what code to replace `CS` with in this case actually isn't contained in the `CS` code itself -- unlike most of the fixable bad codes which are misspellings, or changes in the codes that are used over time, and so you can tell which code it ought to be based just on the observed code.  Whereas here, the information about what code ought to be used instead of `CS` is contextual, and depends on the adjacent records, and that kind of information wouldn't necessarily be applicable to other instances of `CS` that crop up elsewhere.  It's more of a value interpolation / fillna than it is a value substitution.

The more I think about it the more I think not using the encoder is probably the right thing to do.  Ideally we would have another more general process for filling in missing values based on context, but we don't have that yet, so I think @aesharpe's solution is probably good.

Reply:
@zaneselvans yes, that was exactly my thinking.

Reply:
@zaneselvans @zschira  We could still certainly NULL out the `CS` value and then create an interpolation function to fill it in if that feels better.

Reply:
The logic for the interpolation completely makes sense to me, but I'd lean towards nulling it to be consistent with the way we've treated other non-standard codes, and creating an issue to write an interpolation function separately. I've certainly seen other one-off codes that aren't in the historical versions of a form and have nulled them, and it'd be helpful to spell out the interpolation logic a bit more clearly / get a sense of how many values might be affected if we start filling in gaps this way, rather than doing it in a one-off way. Just my two cents!

Reply:
Ok @e-belfer I will null it and link an issue for interpolation.

Do we have a standard way of mapping all values in a column to a set of enum values? I want all instances of `operational_status_code` to adhere to the the codes in `CODE_METADATA["operational_status_eia"]`

Reply:
I think @e-belfer is right that the "correct" thing to do is treat all these code / attribute interpolations the same way. If we have a number of examples of this kind of cleaning needing to be done already and we can easily standardize the process, it would be great to include a list of the known cases in the issue.

Reply:
If you add `CS` to the list of codes which cannot be fixed, it will be NA after encoding step.

Reply:
We don't want to both have an ENUM constraint on the field and a FK relationships, since that would be specifying the same information (about allowable values) in two different ways.

Reply:
When / where does the encoding step happen? Just curious.

Reply:
NM found it.